pkgs/tools/admin/lxd/lxd.nix

```
@@ -0,0 +1,46 @@
+{ lib, hwdata, pkg-config, lxc, buildGoPackage, fetchurl, fetchpatch
+, makeWrapper, acl, rsync, gnutar, xz, btrfs-progs, gzip, dnsmasq, attr
+, squashfsTools, iproute2, iptables, libcap
+, dqlite, raft-canonical, sqlite-replication, udev
+, writeShellScriptBin, apparmor-profiles, apparmor-parser
+, criu
+, bash
+, installShellFiles
+, version, src
+}:
+
+buildGoPackage rec {
+  inherit version src;
+
+  pname = "lxd-unwrapped";
+
+  goPackagePath = "github.com/lxc/lxd";
+
+  postPatch = ''
+    substituteInPlace shared/usbid/load.go \
+      --replace "/usr/share/misc/usb.ids" "${hwdata}/share/hwdata/usb.ids"
+  '';
+
+  preBuild = ''
+    # required for go-dqlite. See: https://github.com/lxc/lxd/pull/8939
+    export CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
+
+    makeFlagsArray+=("-tags libsqlite3")
+  '';
+
+  postInstall = ''
+    installShellCompletion --bash --name lxd go/src/github.com/lxc/lxd/scripts/bash/lxd-client
+  '';
+
+  nativeBuildInputs = [ installShellFiles pkg-config ];
+  buildInputs = [ lxc acl libcap dqlite.dev raft-canonical.dev
+                  sqlite-replication udev.dev ];
```

```suggestion
  nativeBuildInputs = [ installShellFiles pkg-config ];

  buildInputs = [
    lxc
    acl
    libcap
    dqlite.dev
    raft-canonical.dev
    sqlite-replication
    udev.dev
  ];
```

Did you check if we still need `.dev`?